go/worker/compute/executor/committee/node.go

```diff
@@ -523,6 +523,20 @@ func (n *Node) handleScheduleBatch(force bool) { // nolint: gocyclo
 		return
 	}
 
+	// If the next block will be an epoch transition block, do not propose anything as it will be
+	// reverted anyway (since the committee will change).
+	epochState, err := n.commonNode.Consensus.Beacon().GetFutureEpoch(n.roundCtx, n.commonNode.CurrentBlockHeight)
+	if err != nil {
+		n.logger.Error("failed to fetch future epoch state",
+			"err", err,
+		)
+		return
+	}
+	if epochState != nil && epochState.Height == n.commonNode.CurrentBlockHeight+1 {
+		n.logger.Debug("next consensus block is an epoch transition, not scheduling batch")
```

Comment:
```suggestion
		n.logger.Debug("not scheduling a batch, next consensus block is an epoch transition")
```
[nit] Could change to be consistent with other messages:
```
not scheduling a batch, not an executor
not scheduling a batch, incorrect state
not scheduling a batch as we are not a transaction scheduler
```
The last one could also be changed to
```
not scheduling a batch, not a transaction scheduler
```